src/Components/Server/src/Microsoft.AspNetCore.Components.Server.csproj

```diff
@@ -10,6 +10,7 @@
     <DefineConstants>$(DefineConstants);ENABLE_UNSAFE_MSGPACK;SPAN_BUILTIN;MESSAGEPACK_INTERNAL;COMPONENTS_SERVER</DefineConstants>
     <IsPackable>false</IsPackable>
     <EmbeddedFilesManifestFileName>Microsoft.Extensions.FileProviders.Embedded.Manifest.xml</EmbeddedFilesManifestFileName>
+    <Nullable>annotations</Nullable>
```

Comment:
The vast majority of the server's code is internal implementation types and I don't want to spend time fixing that up at this point. Consequently it only has annotations on, which means you get nullability hints, but the code is not checked for correctness.

Reply:
Does this means the compiler doesn't do the check within the library but that we "trust" the annotations?

Reply:
Yup. The API surface is annotated, but the implementation isn't null-checked. I want to do a follow up on these things, but it'd be useful to get the user API in first.